examples/applications/plot_model_complexity_influence.py

```diff
@@ -191,7 +192,7 @@ def _count_nonzero_coefficients(estimator):
     },
     {
         "estimator": GradientBoostingRegressor,
-        "tuned_params": {"loss": "squared_error"},
+        "tuned_params": {},
```

Comment:
I don't understand how this can impact the performance as `GradientBoostingRegressor` already uses the squared error as its default loss.